dateparser_data/supplementary_language_data/date_translation_data/ru.yaml

```diff
@@ -56,7 +59,34 @@ relative-type:
         - послезавтра
 
 simplifications:
-    - ^час: 1 час
-    - ^минуту: 1 минуту
-    - ^секунду: 1 секунду
+    - од(на|ну|ни|ной|ин): '1'
+    - дв(а|е|ое|ух): '2'
+    - пар[ауы]: '2'
+    - три: '3'
+    - четыре: '4'
+    - пять: '5'
+    - шесть: '6'
+    - семь: '7'
+    - восемь: '8'
+    - девять: '9'
+    - десять: '10'
+    - одиннадцать: '11'
+    - двенадцать: '12'
+    - пятнадцать: '15'
+    - двадцать: '20'
+    - тридцать: '30'
+    - сорок: '40'
+    - пятьдесят: '50'
     - несколько секунд: 44 секунды
```

Comment:
wow I'm curious where is this coming from (I know it was here before)

Reply:
Hi @lopuhin, thanks for your comments. I added numbers 1-12 spelled out as words for consistency with other languages (I checked French, German, and English). For round numbers 20-50, I thought it would be useful for referring time e.g. `двадцать минут назад, сорок минут назад (20/40 minutes ago)` which I consider as quite common phrases.

Reply:
Thanks for context, this is useful, and it's great to have this phrases added for Russian as well as for other languages, the choice of words makes sense!

To be clear, I was referring to and old entry of "несколько секунд: 44 секунды" which looks quite strange - but this is not related to the PR.